mon-pix/tests/unit/components/routes/campaigns/skill-review-test.js

```diff
@@ -479,4 +479,55 @@ describe('Unit | component | Campaigns | Evaluation | Skill Review', function()
       expect(result).to.be.true;
     });
   });
+
+  describe('#retryAllowed', function() {
+
+    it('should return false when masteryPercentage equals to 100 ', function() {
+      // given
+      component.args.model.campaignParticipation.isShared = true;
+      component.args.model.campaignParticipation.campaignParticipationResult.masteryPercentage = 100;
+      component.args.model.campaignParticipation.campaign.multipleSendins = true;
+      // when
+      const result = component.retryAllowed;
+
+      // then
+      expect(result).to.be.false;
+    });
+
+    it('should return should return false when isShared is false', function() {
+      // given
+      component.args.model.campaignParticipation.isShared = false;
+      component.args.model.campaignParticipation.campaignParticipationResult.masteryPercentage = 56;
+      component.args.model.campaignParticipation.campaign.multipleSendins = true;
+      // when
+      const result = component.retryAllowed;
+
+      // then
+      expect(result).to.be.false;
+    });
+
+    it('should return false when multipeSendings is false', function() {
+      // given
+      component.args.model.campaignParticipation.isShared = true;
+      component.args.model.campaignParticipation.campaignParticipationResult.masteryPercentage = 56;
+      component.args.model.campaignParticipation.campaign.multipleSendings = false;
+      // when
+      const result = component.retryAllowed;
+
+      // then
+      expect(result).to.be.false;
+    });
+
+    it('should return true when multipeSendings and isShared are true and  masteryPercentage is less than 100', function() {
```

Comment:
typo : manque un L : `should return true when multipLeSendings and isShared are true and masteryPercentage is less than 100`